ads/kixer.js

```diff
@@ -39,11 +44,34 @@ export function kixer(global, data) {
     d.removeEventListener('load', kxload, false);
     if (d.childNodes.length > 0) {
       global.context.renderStart();
+      view_interval = setInterval(function() {
+        kxview_check(); // Once the ad has loaded, start checking for an ad view
```

Comment:
is such a poll necessary? `observeIntersection` will call the callback when there is a change.